### PR TITLE
Docs: Improve Proxmox installation guide

### PR DESCRIPTION
**Pull request by l3rady**
# Pull Request

## What? (description)
Added notes to the proxmox install guide to inform readers of a few gotchas to be aware of

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: website/content/v1.4/talos-guides/install/virtualized-platforms/proxmox.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ Finish creating the VM by clicking through the "Confirm" tab and then "Finish".
 Repeat this process for a second VM to use as a worker node.
 You can also repeat this for additional nodes desired.
 
+> Note: Talos OS doesn't support hot plugging, if creating the VM programmatically don't enable memory hotplug on your
```
Comment (TimJones) [inline website/content/v1.4/talos-guides/install/virtualized-platforms/proxmox.md:108]:
```suggestion
> Note: Talos doesn't support memory hot plugging, if creating the VM programmatically don't enable memory hotplug on your
```

File: website/content/v1.5/talos-guides/install/virtualized-platforms/proxmox.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ Finish creating the VM by clicking through the "Confirm" tab and then "Finish".
 Repeat this process for a second VM to use as a worker node.
 You can also repeat this for additional nodes desired.
 
+> Note: Talos OS doesn't support hot plugging, if creating the VM programmatically don't enable memory hotplug on your
```
Comment (TimJones) [inline website/content/v1.5/talos-guides/install/virtualized-platforms/proxmox.md:108]:
```suggestion
> Note: Talos doesn't support memory hot plugging, if creating the VM programmatically don't enable memory hotplug on your
```

**Comment by smira** [comment]
also might be cool to add https://github.com/siderolabs/talos/issues/7402

File: website/content/v1.5/talos-guides/install/virtualized-platforms/proxmox.md (hunk 2 above)
Comment (l3rady) [reply website/content/v1.5/talos-guides/install/virtualized-platforms/proxmox.md:108]: Thanks for the suggested changes I will get onto that tomorrow. Out of curiosity, is it just memory hotplug that is not supported or is all hot plugging not supported such as CPU, Disk and USB?

Comment (smira) [reply website/content/v1.5/talos-guides/install/virtualized-platforms/proxmox.md:108]: blockdevice/PCI hotplug should definitely work. not sure if Kubernetes supports memory hotplugging (?)

**Review by frezbo — APPROVED** [review]

**Review by TimJones — APPROVED** [review]

**Comment by netthier** [comment]
@smira @l3rady IMO the phrasing used for the #7402 fix is off.
>  Note: As of Talos v1.0 (which requires the x86-64-v2 microarchitecture), prior to Proxmox V8.0, booting with the
> default Processor Type `kvm64` will not work.

To me this seems to imply that starting with Proxmox v8.0 the default processor type 'kvm64' will work.
However, kvm64 will continue to not work with Proxmox using a different default instead ('x86-64-v2-AES').

Sorry for the late comment, but I was only notified just now about this PR existing.

**Pull request merged**